src/main/java/mate/academy/util/HashUtil.java

```diff
@@ -0,0 +1,35 @@
+package mate.academy.util;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+
+public class HashUtil {
+    public static final String CRYPTO_ALGORITHM = "SHA-256";
+
+    private HashUtil() {
+    }
+
+    public static byte[] getSalt() {
+        SecureRandom secureRandom = new SecureRandom();
+        byte[] salt = new byte[8];
+        secureRandom.nextBytes(salt);
+        return salt;
+    }
+
+    public static String hashPassword(String password, byte[] salt) {
+        StringBuilder hashedPassword = new StringBuilder();
+        try {
+            MessageDigest messageDigest = MessageDigest.getInstance(CRYPTO_ALGORITHM);
+            messageDigest.update(salt);
+            byte[] digest = messageDigest.digest(password.getBytes());
+
+            for (byte b : digest) {
+                hashedPassword.append(String.format("%82x", b));

```

Comment:
The format specifier `%82x` seems incorrect for formatting bytes to hex. It should be `%02x` to ensure each byte is represented by two hex digits, which is the standard way to convert bytes to a hex string.

Reply:
ok, done